app/pathogen/arbovirus/dashboard/(map)/MapArbovirusStudySubmissionPrompt.tsx

```diff
@@ -0,0 +1,34 @@
+import { Card, CardContent, CardHeader } from "@/components/ui/card";
+import { X } from "lucide-react";
+import { cn } from "@/lib/utils";
+import Link from "next/link";
+
+interface MapArbovirusStudySubmissionPromptProps {
+  className?: string;
+  onClose: () => void;
+  hidden: boolean;
+}
+
+export const MapArbovirusStudySubmissionPrompt = ({
+  className,
+  onClose,
+  hidden,
+}: MapArbovirusStudySubmissionPromptProps) => {
+  return (
+    <Card className={cn(className, hidden ? "hidden" : undefined)}>
+      <CardHeader className={"p-2 flex flex-row space-y-0"}>
+        <h3 className={"w-full text-center text-lg"}>
+          Don&apos;t see an arbovirus study listed?
```

Comment:
Copy and such pending feedback and subject to change fyi
https://3.basecamp.com/5061858/buckets/30589330/messages/6924228012